draft-ietf-teep-architecture.md

```diff
@@ -459,26 +450,77 @@ all components are further explained in the following paragraphs.
+the rich operating system hosts multiple TEEP brokers, where each broker manages 
+a particular TEE or set of TEEs. Enumeration and access to the appropriate 
+broker is up to the rich OS and the applications. Verification that correct TA 
+has been reached then becomes a matter of properly verifying TA attestations, 
+which are unforgeable. The multiple TEE approach is shown in the diagram below. 
+For brevity, TEEP Broker 2 is shown interacting with only one TAM and UA, but 
+no such limitation is intended to be implied in the architecture. 
+
+~~~~
+   +-------------------------------------------+
+   | Device                                    |
+   |                          +--------+       |        Service Provider
+   |                          |        |----------+               |
+   |    +-------------+       | TEEP   |---------+|               |
+   |    | TEE-1       |<------| Broker |       | ||   +--------+  |
+   |    |             |       | 1      |<---+  | |+-->|        |<-+
+   |    | +---+ +---+ |       |        |    |  | |  +-|  TAM-1 |
+   |    | |TA1| |TA2| |       |        |<-+ |  | +->| |        |<-+
+   |  +-->|   | |   |<---+    +--------+  | |  |    | +--------+  |
+   |  | | +---+ +---+ |  |                | |  |    | TAM-2  |    |
+   |  | |             |  |     +-------+  | |  |    +--------+    |
+   |  | +-------------+  +-----| App-2 |--+ |  |       ^          |
+   |  |                    +-------+   |    |  |       |        Device       
+   |  +--------------------| App-1 |   |    |  |       |   Administrator
+   |                +------|       |   |    |  |       |
+   |    +-----------|-+    |       |---+    |  |       |
+   |    | TEE-2     | |    |       |--------+  |       |
+   |    |           | |    |       |------+    |       |
+   |    | +---+     | |    +-------+      |    |       |
+   |    | |TA3|<----+ |    +----------+   |    |       |
+   |    | |   |       |    | TEEP     |<--+    |       |
+   |    | +---+       |<---| Broker   |----------------+
+   |    |             |    | 2        |        |
+   |    |             |    |          |        |
+   |    +-------------+    +----------+        |
+   |                                           |
+   +-------------------------------------------+
+~~~~
+{: #notionalarch title="Notional Architecture of TEEP wtih multiple TEEs"}
+
+In the diagram above, TEEP Broker 1 controls interactions with the TA's in TEE-1,
+and TEEP Broker 2 contorls interactions with the TA's in TEE-2. This presents some
+challenges for a TAM in completely managing the device, since a TAM may not 
+interact with all the TEEP Broker's on a particular platform. In addition, since
+TEE's may be physically separated, with wholly different resources, there may be no
+need for TEEP Broker's to share information on installed TAs or resource usage.
+However, the architecture guarantees that the TAM will receive all the relevant
+information from the TEEP Broker to which it connects.
```

Comment:
"to which it connects" isn't quite right, since the Broker connects to the TAM, not the other way around.
Either "the TEEP Broker that connects to it", or "the TEEP Broker with which it communicates"

Reply:
agree "to which it communicates" is better